kustomize/base-openshift/shard/cluster-role.yaml

```diff
@@ -27,42 +37,60 @@ rules:
       - secrets
       - configmaps
     verbs:
-      - '*'
-  - apiGroups:
-      - route.openshift.io
-    resources:
-      - routes/custom-host
-    verbs:
+      - get
+      - list
+      - watch
       - create
+      - delete
+      - patch
+      - update
   - apiGroups:
-      - route.openshift.io
+      - networking.k8s.io
     resources:
-      - routes
+      - ingresses
     verbs:
-      - '*'
+      - get
```

Comment:
I guess you had a typo. This cluster role is for OCP you have to keep `route.openshift.io/routes`

Reply:
@r00ta Could I ask why we need to maintain these duplicate file. Can't we directly read them from `src/main/kubernetes/openshift.yml`

Reply:
I mistakenly copy content from minikube file. I had fixed it. Do we have similar `kustomize` file minikube as well?